CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_zuora v0.2.0
+[PR #8](https://github.com/fivetran/dbt_zuora/pull/8) includes the following breaking changes:
+
+## 🚨 Breaking Changes 🚨
+- Redesigned `zuora__billing_history` and `int_zuora__billing_enriched` so `invoice_id` was the actual grain, accounting for the cases when invoices can be tied to multiple credit balance adjustments and payments. This required the following steps:
+    - All metrics associated with `credit_balance_adjustment_id` were aggregated on the invoice level (number of credit balance adjustments,total credit balance amount in home currency, first and last dates of credit balance adjustments on the invoice). Fields associated with an individual `credit_balance_adjustment_id` were removed.
+    - All metrics associated with `payment_id` were aggregated on the invoice level (number of payments, total payment amount in home currency, first and last dates of payments on the invoice). Fields associated with an individual `payment_id` were removed.
+    - Aggregated count of `payment_method_id` to gather a count of payment methods on each invoice.
+- Updated yml of `zuora__billing_history` with aggregated fields, and removed non-aggregated fields associated with credit balance adjustments, payments, and payment methods.
+- Modified `int_zuora__account_enriched` to account for new aggregated metric fields being pulled from `zuora__billing_history`. 
+
+## 🔧 Under The Hood 🔩
+- Updated seed files in `integration_tests` to reproduce the initial `invoice_id` test issue for multiple credit balance adjustments and payments and test resolution. 
```

Comment:
I would just specify here that it was a uniqueness test issue, maybe 

"...uniqueness test issue for multiple credit balance adjustments and payments for one invoice".

Reply:
Updated!